### PR TITLE
fix: flaky test `Click bridge button @no-mmi loads portfolio tab from asset overview when flag is turned off`

### LINKED ISSUE #26437: Fix "Click bridge button from wallet overview @no-mmi loa..." flaky tests
https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/95537/workflows/0b8ca5cb-9f6a-4bd1-b8e8-4dbd359bdb86/jobs/3556379/tests

### PR DESCRIPTION
**Pull request by seaona**
## **Description**

This test is flaky at the step of importing the token and then clicking the asset. Instead of going to the asset page, we remain at the home page. This is likely due to a re-render, but after checking the test steps, we can see how several steps are not needed and we can tweak fixtures to achieve the desired state.
Once using fixtures, the flaky step is removed, so there's no need to fix that, as that step will not occur now.

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/26654?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/26437

## **Manual testing steps**

1. Check ci

## **Screenshots/Recordings**
The test was flaky at the point after importing the token and then clicking the asset (nothing happens). See pic:

image

## **Pre-merge author checklist**

- [x] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: test/e2e/tests/bridge/bridge-test-utils.ts [hunk 1]
```diff
@@ -59,44 +57,12 @@ export class BridgePage {
     );
   };
 
-  navigateToAssetPage = async (
-    contractRegistry: GanacheContractAddressRegistry,
-    symbol: string,
-    shouldImportToken = true,
```
Comment (seaona) [inline test/e2e/tests/bridge/bridge-test-utils.ts:65]: we don't need to import the token manually, we can use fixtures for that. This way, this method sticks to what it's named: just `navigateToAssetPage`

File: test/e2e/tests/bridge/bridge-test-utils.ts [hunk 2]
```diff
@@ -59,44 +57,12 @@ export class BridgePage {
...
-        LOCATOR.MM_IMPORT_TOKENS_MODAL('import-button'),
-      );
-      await this.driver.assertElementNotPresent(
-        '[data-testid="import-tokens-modal-import-button"]',
-      );
-      await this.driver.clickElement({ text: symbol });
-    } else {
-      await this.driver.clickElement({
-        css: '[data-testid="multichain-token-list-button"]',
-        text: symbol,
```
Comment (seaona) [inline test/e2e/tests/bridge/bridge-test-utils.ts:96]: the test was flaky at this point, but now we remove all these steps as they are not needed

File: test/e2e/tests/bridge/bridge-button-opens-portfolio.spec.ts [hunk 3]
```diff
@@ -17,21 +17,20 @@ describe('Click bridge button @no-mmi', function (this: Suite) {
 
   it('loads portfolio tab from asset overview when flag is turned off', async function () {
     await withFixtures(
-      // withErc20 param is false, as we test it manually below
```
Comment (seaona) [inline test/e2e/tests/bridge/bridge-button-opens-portfolio.spec.ts:20]: we support passing a chainId to the `withTokensControllerERC20` method, so now we are able to use fixtures and remove all extra steps

**Review by chloeYue — APPROVED** [review]
LGTM ! Thanks for the fix !

**Review by digiwand — APPROVED** [review]

**Pull request merged**